CONTRIBUTING.md

```diff
@@ -0,0 +1,586 @@
+
+Some preexisting feature filters are configured using an older raw SQL format in `jinja` files in the `queries/` directory. This older syntax is still helpful to select multiple feature properties across many kinds of features at once. For example, [pois.jinja](queries/pois.jinja).
+
+<div class='alert-message'>Generally perform maintenance on pre-existing jinja filters or optionally migrate them to the newer YAML format.</div>
+
+Additionally, the root `queries.yaml` specifies which `jinja` file to use per layer, and also specifies post-processing via Python transforms.
+
+### Serving tiles in the TileServer repo
+
+Listens for API requests on localhost, which are in the format of 0/0/0.ext
+
+When TileServer hears a request it asks Postgres for "the stuff" inside that tile's bounding box, configured via the `queries.yaml` file and `.jinja2` files.
+
+_**NOTE:** A change to one of the query files (jinja) requires TileServer to be restarted so they can be reloaded. But content filter changes (YAML) generally doesn't require restarting TileServer._
+
+
+## Let's do this!
+
+We'll cover the following topics in the next sections:
+
+- [Choose an issue to work on](CONTRIBUTING.md#1-choose-an-issue-to-work-on)
+- [Do your work in a local branch](CONTRIBUTING.md#2-create-a-new-branch)
+- [Create a new test](CONTRIBUTING.md#3-create-a-new-test)
+- [Edit database &/or query logic](CONTRIBUTING.md#4-edit-database-or-query-logic)
+- [Verify the new logic by running the test](CONTRIBUTING.md#5-verify-the-new-logic-by-running-the-test)
+- [Perform any modifications, as necessary](CONTRIBUTING.md#6-perform-any-modifications-as-necessary)
+- [Update data migrations](CONTRIBUTING.md#7-update-data-migrations)
+- [Update documentation](CONTRIBUTING.md#8-update-documentation)
+- [Push your local branch to the server](CONTRIBUTING.md#9-push-your-local-branch-to-the-server)
+- [Submit a Pull Request (PR)](CONTRIBUTING.md#10-submit-a-pull-request-pr)
+
+### 1. Choose an issue to work on
+
+We have a backlog of issues, but they are also grouped into milestones and tracked with [Waffle board](https://waffle.io/tilezen/vector-datasource).
+
+When picking an issue from the Ready column for the active milestone, self assign it to let other people know you'll be working on it and move it to the In Progress column.
+
+If you propose to work on an issue in the Backlog but what to confirm some details add a comment to the issue or ask about it in Slack.
+
+### 2. Create a new branch
+
+Ensure you're on the master branch to establish a clean history:
+
+```bash
+git checkout master
+```
+
+Ensure your `master` branch is up-to-date with the server:
+
+```bash
+git pull
+```
+
+Then checkout a new branch by naming it something descriptive like the `yourname/#issue-issue title`.
```

Comment:
Extra "the"?

Perhaps this would be clearer if it were separated into two sentences? For example:

> You will need a descriptive name for your new branch, and one way to do that is concatenate your user name, any relevant issue number(s) and a brief description, for example `olga/875-camp-ground-zoom`. Then create a new branch using `git checkout -b branchname`, like so: